web/src/routes/settings/debug/+page.svelte

```diff
@@ -15,38 +38,64 @@
 </script>
 
 {#if $settings.advanced.debug}
-    <div id="advanced-page">
-        <h3>device:</h3>
-        <div class="message-container subtext">
-            {JSON.stringify(device, null, 2)}
-        </div>
-
-        <h3>app:</h3>
-        <div class="message-container subtext">
-            {JSON.stringify(app, null, 2)}
-        </div>
-
-        <h3>settings:</h3>
-        <div class="message-container subtext">
-            {JSON.stringify($storedSettings, null, 2)}
-        </div>
-
-        <h3>version:</h3>
-        <div class="message-container subtext">
-            {JSON.stringify($version, null, 2)}
-        </div>
+    <div id="debug-page">
+        {#each sections as { title, data }, i}
+            <div id="debug-section-title">
+                <h3>{title}</h3>
+
+                <button
+                    id="debug-section-copy-button"
+                    aria-label={lastCopiedSection === i
+                        ? $t("button.copied")
+                        : $t("button.copy")}
+                    on:click={() => {
+                        clearTimeout(lastCopiedSectionResetTimeout);
+                        lastCopiedSection = i;
+                        copyURL(JSON.stringify(data, null, 2));
```

Comment:
could you make it copy a compressed version of json? just remove extra params from `JSON.stringify()`

Reply:
If this is meant to be copied + pasted somewhere like Discord, wouldn't we want it to be pretty instead of compressed?